CHANGELOG.md

```diff
@@ -4,10 +4,23 @@ All notable changes to this project will be documented in this file.
 
 Please keep one empty line before and after all headers. (This is required for `git` to produce a conflict when a release is made while a PR is open and the PR's changelog entry would go into the wrong section).
 
+    /// If true, this will ignore diacritical marks and accent characters from
+    /// If true, this will ignore diacritical marks and accent characters from
+    /// If true, this will ignore diacritical marks and accent characters from
+    /// being processed as received characters. Instead, the input
+    /// device's raw character will be placed in event queues with the
+    /// Alt modifier set.
+    /// being processed as received characters. Instead, the input
+    /// device's raw character will be placed in event queues with the
+    /// Alt modifier set.
+    /// being processed as received characters. Instead, the input
+    /// device's raw character will be placed in event queues with the
+    /// Alt modifier set.
 And please only add new entries to the top of this list, right below the `# Unreleased` header.
 
 # Unreleased
-
+<<<<<<< HEAD
```

Comment:
This doesn't look right.

Reply:
sorry, missed when i was updating the branch to master.

Reply:
What's the best way of finding out the mac maintainers?

Reply:
https://github.com/rust-windowing/winit/wiki/Testers-and-Contributors

It's @madsmtm.

Reply:
Thanks!